### ISSUE TITLE
No ore dubling with create

### ISSUE THREAD

**Issue by AlemSnyder** [MrAwesomeRocks/the-latest-summer-fun#26]
create ore doubling has been removed.

**Comment by AlemSnyder** [comment]
https://github.com/MrAwesomeRocks/the-latest-summer-fun/blob/fe03aa0b9fbf3477ebd4367f1b2d55d9acf2656d/modpack/overrides/kubejs/server_scripts/theLatestSummerFun/recipes/block.js#L25

This line in particular.

**Comment by egelja** [comment]
> https://github.com/MrAwesomeRocks/the-latest-summer-fun/blob/fe03aa0b9fbf3477ebd4367f1b2d55d9acf2656d/modpack/overrides/kubejs/server_scripts/theLatestSummerFun/recipes/block.js#L25
> 
> This line in particular.

https://github.com/MrAwesomeRocks/the-latest-summer-fun/blob/fe03aa0b9fbf3477ebd4367f1b2d55d9acf2656d/modpack/overrides/kubejs/server_scripts/theLatestSummerFun/recipes/block.js#L15

This only adds recipes for ingots and gems, and a milling, not a crushing recipe.

All ore doubling works in single-player.

**Comment by AlemSnyder** [comment]
> This only adds recipes for ingots and gems, and a milling, not a crushing recipe.

Yes that is not the line

> All ore doubling works in single-player.

No?
![Screenshot (5)](https://user-images.githubusercontent.com/44094752/129270263-46eb1a1a-c271-4c38-a453-f7601bbb9217.png)

**Comment by egelja** [comment]
Hit F3+H, which mod adds the recipe?

**Comment by AlemSnyder** [comment]
Create

**Comment by AlemSnyder** [comment]
I think you overwrote all the crushing recipes with milling recipes.

**Comment by AlemSnyder** [comment]
IE crushing has been gutted, so has thermal pulverizing. Induction smelting has also been removed for all intents and purposes.

**Comment by egelja** [comment]
And what is the recipe ID? Hit shift while hovering to see it.

**Comment by AlemSnyder** [comment]
>I think you overwrote all the crushing recipes with milling recipes.

Yup they are all milling recipes. There are some that are not milling recipes, they were added by kjs in 

`the-latest-summer-fun/modpack/overrides/kubejs/server_scripts/theLatestSummerFun/recipes/block.js`

Two example recipe IDs:

- `create:kjs_1goubo5mcg5sqvxq5b1h3oc3l`
- `create:milling/iron_ore`
